main/test/ca/uwaterloo/flix/language/phase/TestParser.scala

```diff
@@ -731,6 +731,87 @@ class TestParser extends FunSuite with TestUtils {
     run(input)
   }
 
+  test("Expression.Array.01") {
```

Comment:
Embedding tests inside Scala test is deprecated. We only do that for "negative tests", i.e. tests that should fail in the future.

Instead you should create tests files like:

- Test.Expression.ArrayLit.flix
- Test.Expression.ArrayLoad.flix
- Test.Expression.ArrayStore.flix

and so on.